### DISCUSSION TITLE
معرفی کانتینرهای ایرانی یا نزدیک به ایران

### DISCUSSION THREAD

**Discussion by parsatavafi** [hiddify/Hiddify-Manager#431] [General سایر]
>  روش دوم: استفاده از ابرک‌ها container
> 
> 
> برای بالاآوردن relay داخل  [هم‌روش](https://hamravesh.com) از این روش می‌تونید استفاده کنید. امیدوارم مفید باشه.
> 
> روش بالاآوردن برای هم‌روش:
> 1- ساخت اپلیکیشن داخل پنل (میتونید برای بار اول از اون اعتبار رایگان اولیه که موقع ثبت‌نام میده استفاده کنید و هزینه‌ای پرداخت نکنید)
> 2- انتخاب داکر ایمیج به عنوان نوع اپلیکیشن
> 3- نوشتن hpello/tcp-proxy به عنوان ایمیج و latest به عنوان تگ.
> 4- نوشتن نام اپ
> 5- نوشتن `tcp-proxy <server-ip> 443` به عنوان `دستور اجرایی`
> 6- به داخل تب پورت‌ها برید و یک پورت با اسم main (حتما همین اسم) و پورت داخلی و خارجی 443 درست بکنید.
> 7- به تب آدرس دامنه برید و گزینه ساخت گواهی SSL‍ رو انتخاب کنید و اگر گزینه احراز هویت فعال هست اون رو غیر فعال کنید. یک دامنه بسازید (ترجیحا با خود زیردامنه darkube.app)
> 8-انتخاب پلن رو بزنید و دیتاسنتر مورد نظر رو انتخاب کنید. (دقت کنید برای منابع من خودم برای حدود ۱۰ الی ۲۰ نفر هست سرورم و 250MB و 250m cpu انتخاب کردم.
> 9- یکم صبر کنید که تغییرات DNS و پورت اعمال بشه (حدود ۴-۵ دقیقه)
> 10- داخل پنل ادمین hiddify (دم توسعه دهندگانش گرم) دامنه ای که توی مرحله 7 انتخاب کردید رو بزنید به عنوان CDN و آماده استفاده میشه.
> 
> با این روش من تست کردم و ws اوکی بود. ظاهرا tls+tcp مشکل داشت و ارور 404 میگیره
> 
> _Originally posted by @smf8 in https://github.com/hiddify/hiddify-config/discussions/129#discussioncomment-4996908_

      
روشی که با استفاده از داکر و ایمپورت ایمیجی که دوست عزیزمون گفتند بسیار خوب بود. منتها `هم روش` دسترسی به سرویس رو میبنده یعنی من حدود دو هفته داشتم از روشی که دوستمون گفتند استفاده می کردم و بعدش اکانت رو بن کردند، تماس گرفتند و بسیار محترمانه گفتند فعلا نمیتونیم سرویسی برای این کار به شما بدیم. من خواهشی که از دوستان و اساتید دارم اینه که اگر امکان داره شرکت هایی که به عنوان تونل میتونیم ازشون سرویس بگیریم رو معرفی کنید اگر با روش کانتینر و داکر باشه که نصب و تنظیمش هم در کسری از ثانیه انجام میشه! اگر نه با هر روشی که بشه تونل ایجاد کرد خیلی از مشکلات رو حل می کنه.

**Comment by salaryy** [comment]
دقیقا درست میفرمایید سرویس بدون cdn تقریبا هر تونلی بی فایدست 
روش درتسش cloudflare CDN هست که اونم تقریبا فیلتر شده
الان فقط سرویس های CDN رو باید پیدا کرد که قابلیت data transfer رو داشته باشه
